package.json

```diff
@@ -7,8 +7,8 @@
     "build": "npm run build --workspaces",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "lint": "eslint packages/** --ext .ts",
-    "lint:fix": "eslint --fix packages/** --ext .ts"
+    "lint": "find packages/ -name '*.ts' | grep -q . && eslint packages/** --ext .ts || echo 'No matching files'",
+    "lint:fix": "find packages/ -name '*.ts' | grep -q . && eslint --fix packages/** --ext .ts || echo 'No matching files'"
```

Comment:
This is getting really complicated. Can you try with this alternative?
```bash
eslint packages/** --ext .ts --no-error-on-unmatched-pattern
```

Reply:
okay